internal/policy/helpers_test.go

```diff
@@ -148,8 +148,9 @@ func createTestStateWithPolicy(t *testing.T) *State {
 	}
 
 	return &State{
-		RootEnvelope:    rootEnv,
-		TargetsEnvelope: targetsEnv,
-		RootPublicKeys:  []*tuf.Key{key},
+		RootEnvelope:        rootEnv,
+		TargetsEnvelope:     targetsEnv,
+		DelegationEnvelopes: map[string]*sslibdsse.Envelope{}, // FIXME: this isn't the best fix. Instead, LoadState* methods should set this to nil when no delegated roles exist.
```

Comment:
Is this something we want to track with an issue, or nah?

Reply:
I plan to submit a quick patch fixing these up after this is merged.